zerver/actions/invites.py

```diff
@@ -34,7 +35,7 @@
 def notify_invites_changed(realm: Realm) -> None:
     event = dict(type="invites_changed")
     admin_ids = [user.id for user in realm.get_admin_users_and_bots()]
-    send_event(realm, event, admin_ids)
+    transaction.on_commit(lambda: send_event(realm, event, admin_ids))
```

Comment:
I feel like we already have a bunch of things like this, and are going to add more (like this) -- all of which are hust deferred calls to `send_event` (or `queue_json_publish`).  We should make a helper function of `@transaction.atomic(savepoint=False) def deferred_send_event(...)` (and another for `deferred_queue_publish`) which does this for us.